libs/ui/src/components/atoms/helperTexts/helperTexts.tsx

```diff
@@ -57,3 +67,17 @@ export const ErrorText = ({ children }: SpecialHelperText) => {
     </HelperText>
   );
 };
+
+export const HelperTextFactory = ({
+  error,
+  success,
+  warning,
+  helperText,
+}: HelperTextFactoryProps) => {
+  if (!error && !success && !warning && !helperText) return null;
```

Comment:
Once the above params are change we can do this check on an object of values rather than having a long boolean statement